wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,75 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+A user may get banned if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Circumventing a punishment by making a new account.
+-   Sharing their account with other users.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
 
-What is considered cheating?
-----------------------------
 
-As the rules says, any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
+## What are considered cheating?
+The rules say that any form of manipulation of the client such as score hacking, macro-enabled full-combo sequences, auto-aim and anything else that plays the game for the player.
 
-This is a one-strike rule; cheaters will be **permanently banned.**
+This is a one-strike rule. Cheaters will be banned on their first offence for an indefinite amount of time. The team may decide to lift their ban if they successfully appeal.
 
-I have been banned for cheating. I didn't cheat!
-------------------------------------------------
 
-osu! uses an automated metrics combined with manual confirmation.
+## I have been banned for cheating. I didn't cheat!
+osu! uses automated metrics combined with manual confirmation.
 
-The staff never places an incorrect ban for cheating. Before they do any actions, they make sure it first 100%.
+The team never places an incorrect ban for cheating. Before they take any action, they make sure it is 100% certain that you cheated.
```

Comment:
we have and that is precisely why bans no longer happen - restrictions are also used as a stopgap measure to prevent further disruption while we investigate a suspicious player